scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -1429,6 +1446,70 @@ while true; do
     fi
   fi
 
+      # Mithril Signer Section
+      mithrilSignerVars
+
+      if [[ "$nodemode" = "Core" ]] && [[ "$serviceExists" -eq 0 ]] && [[ "$signerServiceStatus" == "running" ]] && [[ "$metricsEnabled" == "true" ]]; then
+
+        printf "${mithrildivider}\n" && ((line++))
+
+        get_metric_value() {
+            local metric_name="$1"
+            local metric_value
+            while IFS= read -r line; do
+                if [[ $line =~ $metric_name[[:space:]]+([0-9]+) ]]; then
```

Comment:
This will resolve the linter/checks in the PR showing red, removing the shellcheck error:

```suggestion
                if [[ $line =~ ${metric_name}[[:space:]]+([0-9]+) ]]; then
```